ccm_web/client/src/components/CustomTable.tsx

```diff
@@ -56,10 +56,9 @@ function CustomTable<T extends TableEntity> (props: TableProps<T>): JSX.Element
             return (
               <TableRow hover key={row.rowNumber}>
                 {columns.map((column) => {
-                  const value = row[column.id]
                   return (
                     <TableCell key={String(column.id)} align={column.align}>
-                      {value}
```

Comment:
This change is already in feature branch this change should not have reflected here. I feel there is a problem in the merge/rebase from feature branch